fifteen_min/Cargo.toml

```diff
@@ -21,5 +21,7 @@ getrandom = { version = "0.2.2", optional = true }
 log = "0.4"
 map_gui = { path = "../map_gui" }
 map_model = { path = "../map_model" }
+strum = "0.20"
```

Comment:
You can remove the dependencies here. None of the code in this crate directly uses `strum`; it's all within `map_model`